packages/encryption/src/cryptoUtils.ts

```diff
@@ -0,0 +1,57 @@
+import * as crypto from "crypto";
+
+import { Buffer } from "buffer";
+
+
+interface EncryptedData {
+
+  initialVector:string,
+  encrypted:string,
+  authTag:string
+}
+
+
+export const encrypt=(text:string,key:Buffer):EncryptedData=>{
```

Comment:
Avoid using this kind of arrow function (it looks strange). You can see how we used to write functions. Additionally, make sure to follow code rabbit's suggestions.